Simplenote/NoticeController.swift

```diff
@@ -16,14 +16,15 @@ class NoticeController {
         }
     }
 
-    private var isPresenting: Bool {
+    var isPresenting: Bool {
```

Comment:
This can stay private :-)

Reply:
I merged some changes from the branch that I think cause some properties to change in ways I had not intended....  Fixed